bin/sts-assume-role.sh

```diff
@@ -10,10 +10,11 @@ fi
 
 role_arn="$1"
 region="$2"
+duration"$3"
```

Comment:
There's a missing = here. I think I'd probably handle the default here and have `duration=${3:=1800}` using the := instead of :-. They do similar things but :- makes it look like the default is -1800s.